modules/installation-gcp-user-infra-adding-ingress.adoc

```diff
@@ -34,23 +44,31 @@ NAME             TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)
 router-default   LoadBalancer   172.30.18.154   35.233.157.184   80:32288/TCP,443:31215/TCP   98
 ----
 
-. Add the A record to your public and private zones:
+. Add the A record to your zones:
+** To use A records:
+... Export the variable for the router IP address:
 +
 ----
 $ export ROUTER_IP=`oc -n openshift-ingress get service router-default --no-headers | awk '{print $4}'`
-
-$ if [ -f transaction.yaml ]; then rm transaction.yaml; fi
-$ gcloud dns record-sets transaction start --zone ${BASE_DOMAIN_ZONE_NAME}
-$ gcloud dns record-sets transaction add ${ROUTER_IP} --name \*.apps.${CLUSTER_NAME}.${BASE_DOMAIN}. --ttl 300 --type A --zone ${BASE_DOMAIN_ZONE_NAME}
-$ gcloud dns record-sets transaction execute --zone ${BASE_DOMAIN_ZONE_NAME}
-
+----
+... Add the A record to the private zones:
++
+----
 $ if [ -f transaction.yaml ]; then rm transaction.yaml; fi
 $ gcloud dns record-sets transaction start --zone ${INFRA_ID}-private-zone
 $ gcloud dns record-sets transaction add ${ROUTER_IP} --name \*.apps.${CLUSTER_NAME}.${BASE_DOMAIN}. --ttl 300 --type A --zone ${INFRA_ID}-private-zone
 $ gcloud dns record-sets transaction execute --zone ${INFRA_ID}-private-zone
 ----
+... For an external cluster, add the A record to the public zones:
 +
-If you prefer to add explicit domains instead of using a wildcard, you can
+----
+$ if [ -f transaction.yaml ]; then rm transaction.yaml; fi
+$ gcloud dns record-sets transaction start --zone ${BASE_DOMAIN_ZONE_NAME}
+$ gcloud dns record-sets transaction add ${ROUTER_IP} --name \*.apps.${CLUSTER_NAME}.${BASE_DOMAIN}. --ttl 300 --type A --zone ${BASE_DOMAIN_ZONE_NAME}
+$ gcloud dns record-sets transaction execute --zone ${BASE_DOMAIN_ZONE_NAME}
```

Comment:
Missing --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT} to these command lines.